src/pages/dataSetsWip/form/dataSetFormSchema.ts

```diff
@@ -10,10 +14,13 @@ export const dataSetFormSchema = identifiable
     .extend({
         id: z.string().optional(),
         code: z.string().trim().optional(),
-        description: z.string().trim().optional(),
+        description: z.string().trim().max(2000).optional(),
         style,
         dataElements: referenceCollection.default([]),
-        // categoryCombo: z.object({ id: z.string() }),
+        categoryCombo: z
+            .object({ id: z.string() })
+            .default({ id: DEFAULT_CATEGORY_COMBO.id }),
```

Comment:
Could we change this so we can default to the whole of `default`? Eg so we don't have to fetch `displayName`?

```
categoryCombo: z.object({ id: z.string(), displayName: z.string()}).default({ ...fromdefaultcatcombo})
```

Reply:
I went for your first suggestion as i think really transform is a function that should run once the values are fetched. We could add a default value prop but maybe for now giving the default prop values in their completed form is less complex